### DISCUSSION TITLE
openvino_2024.3.0-2_all.deb

### DISCUSSION THREAD

**Discussion by Superkyrpto** [intel/openvino-plugins-ai-audacity#263] [Q&A]
Ok I hav the deb file for open vino plugins how to I put it in audacity on linux

**Comment by RyanMetcalfeInt8** [comment]
Hello, you'll need to build these plugins for your Linux distro, following these instructions: https://github.com/intel/openvino-plugins-ai-audacity/blob/main/doc/build_doc/linux/README.md

Note that those instructions are specific to Ubuntu 22.04 -- Other distro's may be slightly different.

**Comment by RyanMetcalfeInt8** [comment]
Also, it looks like you have a .deb for OpenVINO itself, not these plugins (OpenVINO is the framework for which these plugins are based on).
